cmd/helm/install.go

```diff
@@ -223,3 +232,89 @@ func isChartInstallable(ch *chart.Chart) (bool, error) {
 	}
 	return false, errors.Errorf("%s charts are not installable", ch.Metadata.Type)
 }
+
+type ValueOptions struct {
```

Comment:
Is there a way we can add this back to the action package? I imagine other users would want to merge values together similar to the CLI, so it'd be great to make this functionality public but optional.

Reply:
Hmm, moving it to `pkg/cli` make sense to me, but it causes an import cycle.

```
import cycle not allowed
package helm.sh/helm/cmd/helm
	imports helm.sh/helm/cmd/helm/search
	imports helm.sh/helm/pkg/repo
	imports helm.sh/helm/pkg/getter
	imports helm.sh/helm/pkg/cli
	imports helm.sh/helm/pkg/getter
```

The `readFile` function used by `MergeValues`  requires importing `pkg/getter`, but `getter` imports `pkg/cli` to use `cli.EnvSettings`.

Not sure if you have any ideas for good ways to resolve that, but for now I'll move `cmd/helm.ValueOptions` to `pkg/cli/values.Options`